.devcontainer/devcontainer.json

```diff
@@ -2,13 +2,16 @@
 // https://github.com/microsoft/vscode-dev-containers/tree/master/containers/docker-existing-dockerfile
 {
 	"name": "Dev Docker Environment",
-	"image": "osm:dev",
+	// "image": "osm:dev",
+	"image": "osm-fips:dev",
+	
 
 	// Sets the run context to one level up instead of the .devcontainer folder.
 	"context": "..",
 
 	// Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
-	"dockerFile": "..\\Dockerfile",
+	// "dockerFile": "../Dockerfile",
+	"dockerFile": "../Dockerfile.fips",
```

Comment:
Should we leave the non-FIPS one as the default?

Reply:
Yeah that's probably fair. I don't think anyone's using it, but defaulting to non-FIPS is probably clearer